### PR TITLE
Reduce ambiguity in what types of fixes need issues vs PR

### PR DESCRIPTION
**Pull request by rocketeerbkw**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-jenkins-builder/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Description:

While contributing to `docker-homeassistant` I was asked not to make an issue AND a PR for the same fix but the only reason I did both was because I read the CONTRIBUTING.md thought I should open an issue first. It was only after I made the issue, then I saw PULL_REQUEST_TEMPLATE.md which made it sound like I shouldn't have opened the issue (but still wasn't clear).

This is a small language change that would've been more clear to me.

## Benefits of this PR and context:
I was confused because I read the original text as "If this is a fix for a typo in code OR a fix for documentation in the README please file an issue" instead of "If this is a fix for a typo in code OR a typo in docs...". Since I was fixing the README, I filed an issue first.

I assume this language is to reduce typo related PR spam and I think my changes can make that more clear.

## How Has This Been Tested?
I ran the generator against itself and double checked the `.github/CONTRIBUTING.md` and `.github/PULL_REQUEST_TEMPLATE.md`

## Source / References:
 - https://github.com/linuxserver/docker-homeassistant/issues/11#issuecomment-862055142
 - https://discord.com/channels/354974912613449730/757585807061155840/854967622500089906

### CODE REVIEW

**Comment by Roxedus** [comment]
Thanks, please rebase this to target the monthly branch

File: roles/generate-jenkins/templates/PULL_REQUEST_TEMPLATE.j2 [hunk 1]
```diff
@@ -13,7 +13,7 @@ This image is deprecated. We will not offer support for this image and it will n
 
 <!--- Before submitting a pull request please check the following -->
 
-<!---  If this is a fix for a typo in code or documentation in the README please file an issue and let us sort it out we do not need a PR  -->
+<!---  If this is a fix for a typo (in code, documentation, or the README) please file an issue and let us sort it out we do not need a PR  -->
```
Comment (nemchik) [inline roles/generate-jenkins/templates/PULL_REQUEST_TEMPLATE.j2:16]:
```suggestion
<!---  If this is a fix for a typo (in code, documentation, or the README) please file an issue and let us sort it out. We do not need a PR  -->
```

File: roles/generate-jenkins/templates/CONTRIBUTING.j2 [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 * While contributing make sure to make all your changes before creating a Pull Request, as our pipeline builds each commit after the PR is open.
 * Read, and fill the Pull Request template
-  * If this is a fix for a typo in code or documentation in the README please file an issue
+  * If this is a fix for a typo (in code, documentation, or the README) please file an issue and let us sort it out we do not need a PR
```
Comment (nemchik) [inline roles/generate-jenkins/templates/CONTRIBUTING.j2:7]:
```suggestion
  * If this is a fix for a typo (in code, documentation, or the README) please file an issue and let us sort it out. We do not need a PR
```

**Review by nemchik — CHANGES_REQUESTED** [review]
A few extra grammar adjustments while we're at our.

**Review by Roxedus — APPROVED** [review]

**Review by nemchik — APPROVED** [review]

**Pull request merged**